Make language more precise.

gambiteer opened the pull request.

gambiteer committed:
Make language more precise.

arthurgleckler commented:
Thanks for this pull request.  I'm waiting for a volunteer to make sure that we've caught all such errors before I post an erratum.

gambiteer committed:
More corrections

See the discussion beginning at

https://srfi-email.schemers.org/srfi-141/msg/21379798/

These specific changes are mentioned here:

https://srfi-email.schemers.org/srfi-141/msg/21441006/

and John Cowan's response.
